openslides/core/static/js/core/site.js

```diff
@@ -661,6 +661,83 @@ angular.module('OpenSlidesApp.core.site', [
     }
 ])
 
+/* change:
+ *  Callback if file changes. The one parameter is csv passing the parsed file
+ * config:
+ * - accept: String with extensions: default '.csv .txt'
+ * - encodingOptions: List with encodings. Default ['UTF-8', 'ISO-8859-1']
+ * - parseConfig: a dict passed to PapaParse
+ */
+.directive('csvImport', [
+    function () {
+        return {
+            restrict: 'E',
+            templateUrl: 'static/templates/csv-import.html',
+            scope: {
+                change: '&',
+                config: '=?',
+            },
+            controller: function ($scope, $element, $attrs, $location) {
+                // set config if it is not given
+                if (!$scope.config) {
+                    $scope.config = {};
+                }
+                if (!$scope.config.parseConfig) {
+                    $scope.config.parseConfig = {};
+                }
+
+                $scope.inputElement = angular.element(document.querySelector('#csvFileSelector'));
+
+                // set accept and encoding
+                $scope.accept = $scope.config.accept || '.csv';
+                $scope.encodingOptions = $scope.config.encodingOptions || ['UTF-8'];
+                $scope.encoding = $scope.encodingOptions[0];
+
+                $scope.parse = function () {
+                    var inputElement = $scope.inputElement[0];
+                    if (!inputElement.files.length) {
+                        $scope.change({csv: undefined});
+                        return;
+                    }
+
+                    var reader = new FileReader();
+                    reader.onload = function(onLoadEvent) {
```

Comment:
You can set PapaParse option `header` to false and then just drop the first line in the result callback.